tests/bench/bench_deriver_silent_loss.py

```
@@ -0,0 +1,130 @@
+            ),
+            patch("src.deriver.deriver.emit", side_effect=emitted.append),
+        ):
+            try:
+                await process_representation_tasks_batch(
+                    messages=[_message()],
+                    message_level_configuration=configuration,
+                    observers=["bob"],
+                    observed="alice",
+                    queue_item_message_ids=[1],
+                )
+            except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)